ci/docker/Dockerfile.build.arm

```diff
@@ -64,13 +64,25 @@ RUN curl -o armv6-eabihf--glibc--stable-2020.02-2.tar.bz2 -L https://toolchains.
     rm armv6-eabihf--glibc--stable-2020.02-2.tar.bz2
 ENV CMAKE_TOOLCHAIN_FILE=/usr/local/armv6-eabihf--glibc--stable-2020.02-2/share/buildroot/toolchainfile.cmake
 
-RUN git clone --recursive -b v0.3.9 https://github.com/xianyi/OpenBLAS.git && \
+RUN git clone --recursive -b v0.3.12 https://github.com/xianyi/OpenBLAS.git && \
```

Comment:
any reason to upgrade the openblas in this PR? looks like openblas hasn't been stable for the past few releases.

Reply:
Only as the PR is modifying the file already. Why do you think it's unstable?

Reply:
The following note appeared in the second last patch version.
`NOTE there appear to be several defects in this version unfortunately - this should not be redistributed or used in a production environment`

Reply:
I think it's ok to upgrade as long as there's enough confidence in the new version. up to you

Reply:
The note only applies to 0.3.11. 0.3.12 should be fine. I think we should update to ensure that we do test the newer versions of OpenBLAS (the old version is tested on CentOS). I can open a separate PR for it if you prefer.